### PR TITLE
Fix macos ctrl context menus on mute addon

### LINKED ISSUE #1891: On mac, ctrl clicking the flag brings up context menu instead of muting
**Describe the bug**

On mac, ctrl clicking the flag brings up context menu instead of muting the project.

**To Reproduce**

1. Go to a project.
2. Right click flag.
3. See error

**Expected behavior**

It would mute the project instead of displaying a context menu.
Something like:

> window.addEventListener('contextmenu', function (e) { 
// mute project
e.preventDefault(); 
}, false);

should fix it.

**Screenshots**

Screen Shot 2021-03-15 at 4 26 45 PM
Screen Shot 2021-03-15 at 4 27 16 PM

**Device (please complete the following information):**

- MacOS Big Sure
- Chrome (Brave) 88

>

### PR DESCRIPTION
**Pull request by GrahamSH-LLK**
**Resolves**

Resolves #1891

**Changes**

Fixes bug
**Reason for changes**

Bugs
**Tests**

Untested

### CODE REVIEW

**Comment by scratchusernamemrtbts** [comment]
i test and it doesn't work

**Comment by GrahamSH-LLK** [comment]
Test now, we just fixed a bug.

**Comment by GrahamSH-LLK** [comment]
@scratchusernamemrtbts

**Comment by scratchusernamemrtbts** [comment]
Ok

**Comment by scratchusernamemrtbts** [comment]
I'll test later
I need to go to bed now
It's 21.23 at Thailand now

**Comment by scratchusernamemrtbts** [comment]
it works but if you hold the control key for a while it will show the context menu

https://user-images.githubusercontent.com/66320961/111854396-a8a9f900-8951-11eb-85fa-5df38e6177d8.mov

**Review by scratchusernamemrtbts — APPROVED** [review]

**Comment by TheColaber** [comment]
You sure you are running this branch and not the master?

**Comment by scratchusernamemrtbts** [comment]
yes

**Review by TheColaber — APPROVED** [review]

**Comment by scratchusernamemrtbts** [comment]
@Hans5958 merge?

**Comment by Hans5958** [comment]
Not my part here. @WorldLanguages

File: addons/mute-project/userscript.js [hunk 1]
```diff
@@ -4,12 +4,26 @@ export default async function ({ addon, global, console }) {
...
+  let ctrlPressedRecently = false;
+
+  window.addEventListener("keydown", (event) => {
+    if (event.ctrlKey) {
+      ctrlPressesCount++;
+      const pressCount = ctrlPressesCount;
+      ctrlPressedRecently = true;
+      setTimeout(() => {
+        if (pressCount === ctrlPressesCount) ctrlPressedRecently = false;
+      }, 2500);
```
Comment (WorldLanguages) [inline addons/mute-project/userscript.js:17]: We might want to lower this to less than 2.5 seconds, but in that case Alt+GreenFlag for 60FPS should also be updated, so better to do in a separate PR if someone is interested.

**Pull request merged**